IntuitiveWebSolutions/gen3-template
# Implement `remove:cypress` npm script

This PR implements `remove:cypress` command. It also extracts `constants.js` from `integration` to `tests` directory and updates the readmes accordingly.

Also refactors the `eject` script for consistency.

### Testing steps

- Run `yarn remove:cypress` and answer the prompt with yes.
- Ensure the script runs without any errors.
- Run `git status` and make sure the changes are made.
- Run `git checkout -- .` to restore the index to HEAD.
